src/components/shareable-url-modal/shareable-url-modal.scss

```diff
@@ -275,6 +330,75 @@
   }
 }
 
+.shareable-url-modal__published-wrapper {
+  .modal__wrapper {
+    padding: 0;
+    width: 707px;
+  }
+}
+
+.shareable-url-modal__published {
+  // check with Step since there's no timestamp atm, can we decrease the padding bottom from 84 to 48px
+  background-color: var(--color-modal-bg-1);
+  padding: 48px;
+  width: 100%;
+
+  .shareable-url-modal__content-title {
+    padding-bottom: 32px;
+  }
+}
+
+.shareable-url-modal__published-dropdown-wrapper {
+  display: flex;
+  flex-direction: row;
+
+  .dropdown__label {
+    height: 100%;
+    width: 209px;
+  }
+}
+
+.shareable-url-modal__published-action {
+  align-items: baseline;
+  background: var(--color-modal-bg-1);
+  border-top: 1px solid var(--color-footer-border);
+  display: flex;
+  justify-content: space-between;
+  padding: 48px 48px 56px;
+  width: 100%;
+
+  .button__btn--secondary {
+    // double check the hex colour with Step
```

Comment:
Please remove comment if not needed.

Reply:
I'll leave the comment here so when Stephanie is QA'ing the code I can remember to check with her. Will remove before merging